commands/climber/fullextend.py

```diff
@@ -0,0 +1,71 @@
+from commands2 import CommandBase
+from wpilib import RobotBase
+from subsystems.climbers.leftclimbersubsystem import LeftClimber
+from subsystems.climbers.rightclimbersubsystem import RightClimber
+import constants
+
+
+class FullLeftClimber(CommandBase):
```

Comment:
That isn't quite accurate because this is the same position we would use if we were to climb higher than middle rung too. It's just our full extension

Reply:
We may need to use a different command/extension to get the high bar, depending on the sequencing.

Reply:
Why? A full extension would get us to it most reliably. I don't see what reason we have to go less than full other than maybe a slight speed increase

Reply:
There may be several reasons for the position to be different (frame perimeter considerations, weight management, etc) or it may end up being the same position. The important this is that we are commanding a position that is meaningful in game-terms (capture middle rung) rather than one that is a hardware/subsystem implementation detail (move the drum motor to position foo).